pkg/limayaml/defaults.go

```diff
@@ -81,6 +81,24 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 		}
 	}
 
+	if y.CPUType == nil {
+		y.CPUType = d.CPUType
+	}
+	if o.CPUType != nil {
+		y.CPUType = o.CPUType
+	}
+	if y.CPUType == nil || *y.CPUType == "" {
+		if IsNativeArch(*y.Arch) {
+			y.CPUType = pointer.String("host")
+		} else if *y.Arch == X8664 {
+			// Intel on ARM
+			y.CPUType = pointer.String("qemu64")
+		} else {
+			// ARM on Intel
+			y.CPUType = pointer.String("cortex-a72")
+		}
```

Comment:
If we have specific reasons for picking these CPU types, could you add them in the comments?

Reply:
Sure, or we need additional documentation to explain.

Reply:
I would prefer some explanation in the comments that answers "Why have these values been chosen as the defaults?"

If we have additional documentation, it should expand on this and also offer which alternative values you might want to chose, and why.

I'm not sure if we actually do have the answers to either of these questions, but if we do, it would be nice to record them.